apps/nerves_hub_web_core/lib/nerves_hub_web_core/firmwares.ex

```diff
@@ -122,18 +122,26 @@ defmodule NervesHubWebCore.Firmwares do
   end
 
   def delete_firmware(%Firmware{} = firmware) do
+    do_delete = fn ->
```

Comment:
```suggestion
    do_delete_firmware = fn ->
```

The with statement below doesn't match here and instead references `do_delete_firmware.()`. I like that better, so prob just change here

Reply:
guess i didn't run test before my last push. whoops.